[2024-06-09 14:30:37,198] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_new_data.transform 2024-06-08T00:00:00+00:00 [queued]>
[2024-06-09 14:30:37,205] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: get_new_data.transform 2024-06-08T00:00:00+00:00 [queued]>
[2024-06-09 14:30:37,206] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-09 14:30:37,206] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-09 14:30:37,206] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-09 14:30:37,215] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): transform> on 2024-06-08T00:00:00+00:00
[2024-06-09 14:30:37,220] {standard_task_runner.py:52} INFO - Started process 224 to run task
[2024-06-09 14:30:37,223] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'get_new_data', 'transform', '2024-06-08T00:00:00+00:00', '--job-id', '86', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/get_new_data.py', '--cfg-path', '/tmp/tmpn0wyaz_v', '--error-file', '/tmp/tmpqn099ww2']
[2024-06-09 14:30:37,226] {standard_task_runner.py:77} INFO - Job 86: Subtask transform
[2024-06-09 14:30:37,269] {logging_mixin.py:104} INFO - Running <TaskInstance: get_new_data.transform 2024-06-08T00:00:00+00:00 [running]> on host 50f6ca71452e
[2024-06-09 14:30:37,300] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_new_data
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2024-06-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-08T00:00:00+00:00
[2024-06-09 14:30:39,385] {python.py:118} INFO - Done. Returned value was: None
[2024-06-09 14:30:39,390] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=get_new_data, task_id=transform, execution_date=20240608T000000, start_date=20240609T143037, end_date=20240609T143039
[2024-06-09 14:30:39,407] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-09 14:30:39,442] {local_task_job.py:146} INFO - Task exited with return code 0
